src/build/env.py

```diff
@@ -260,7 +260,12 @@ def _create_isolated_env_venv(path: str) -> Tuple[str, str]:
 
     import packaging.version
 
-    venv.EnvBuilder(with_pip=True, symlinks=_fs_supports_symlink()).create(path)
+    simlinks = _fs_supports_symlink()
```

Comment:
Moved here to make --showlocals more useful in pytest.

Reply:
Typo, should be symlinks.